distro/runtime/concierge/smarthome.xargs

```diff
@@ -3,7 +3,7 @@
 # Version configuration
 -Dconcierge.version=5.0.0
 -Djetty.version=9.3.10
--Desh.version=0.9.0.b1
+-Desh.version=0.9.0-SNAPSHOT
```

Comment:
We used b1 on purpose to avoid a moving target; I'd rather like to update the sample distro whenever a new beta build of ESH is done. Otherwise we risk that the sample distro is broken as it might miss newly added bundles etc. and it will be hard to notice that.